LibreNMS/OS/Airos.php

```diff
@@ -158,9 +158,11 @@ public function discoverWirelessPower()
      */
     public function discoverWirelessQuality()
     {
-        $oid = '.1.3.6.1.4.1.41112.1.4.6.1.3.1'; //UBNT-AirMAX-MIB::ubntAirMaxQuality.1
+        //$oid = '.1.3.6.1.4.1.41112.1.4.6.1.3.1'; //OLD UBNT-AirMAX-MIB::ubntAirMaxQuality.1
```

Comment:
instead of removing the old AirMaxQuality, you should try to discover two sensors here.  Only ones that exist will be added.